templates/taxi/index.html

```diff
@@ -10,4 +10,5 @@ <h2>Dynamic content</h2>
     <li><strong>Drivers:</strong> {{ num_drivers }}</li>
     <li><strong>Manufacturers:</strong> {{ num_manufacturers }}</li>
   </ul>
+    <p>You have visited this page {{ num_visits }} time{{num_visits|pluralize}}</p>
```

Comment:
Ensure that `num_visits` works as expected. When you visit the page for the first time, it should display: `You have visited this page 1 time`.